README.md

```diff
@@ -1,19 +1,33 @@
 # SnarkyJS &nbsp; [![npm version](https://img.shields.io/npm/v/snarkyjs.svg?style=flat)](https://www.npmjs.com/package/snarkyjs) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md)
 
-The easiest way to write zk programs is using SnarkyJS, a TypeScript framework for zk-SNARKs and zkApps.
+The SnarkyJS SDK helps developers build apps powered by zero-knowledge cryptography.
```

Comment:
```suggestion
SnarkyJS helps developers build apps powered by zero-knowledge cryptography.
```
I wouldn't use SDK here, doesn't sound right and it's clear from context that it's a JS library